lib/bulbs-elements/settings.js

```diff
@@ -0,0 +1,31 @@
+let settings = {
+  base: {
+    BULBS_ELEMENTS_ONIONSTUDIOS_GA_ID: 'UA-223393-14',
+    BULBS_ELEMENTS_COMSCORE_ID: '6036328',
+    RESP: 'vmap1',
+  },
+  development: {
+    FREEWHEEL_NETWORK_ID: '111976',
```

Comment:
if the end goal is to have all the settings in one `settings.py` file and we are pushing these settings to `window`. I'd argue, it might make sense to just put them in each sites' `settings.py` file. Having a `settings.js` and `settings.py` file just seems to add an unnecessary and possibly confusing extra layer of complication.